deploy/examples/volume-replication-class.yaml

```
@@ -9,4 +9,4 @@ spec:
     schedulingInterval: "12m"
     schedulingStartTime: "16:18:43"
     replication.storage.openshift.io/replication-secret-name: rook-csi-rbd-provisioner
-    replication.storage.openshift.io/replication-secret-namespace: rook-ceph
+    replication.storage.openshift.io/replication-secret-namespace: rook-ceph # namespace:cluster
```

@Rakshith-R This is the csi driver namespace, right? So it would be the `operator` tag. 
```suggestion
    replication.storage.openshift.io/replication-secret-namespace: rook-ceph # namespace:operator
```